pymc/logprob/order.py

```diff
@@ -49,6 +49,8 @@
 from pytensor.tensor.random.op import RandomVariable
 from pytensor.tensor.var import TensorVariable
 
+import pymc as pm
```

Comment:
If it's just for `pm.math.logdiffexp`, maybe something like `from pymc.math import logdiffexp`?